examples/x86_64.swiftinterface

```diff
@@ -0,0 +1,511 @@
+  public var dispatchQueue: Dispatch.DispatchQueue?
+  public var context: AzureCore.PipelineContext?
+  public init(repeatabilityRequestId: Swift.String? = nil, clientRequestId: Swift.String? = nil, cancellationToken: AzureCore.CancellationToken? = nil, dispatchQueue: Dispatch.DispatchQueue? = nil, context: AzureCore.PipelineContext? = nil)
+}
+public struct UpdateChatMessageOptions : AzureCore.RequestOptions {
+  public let clientRequestId: Swift.String?
+  public let cancellationToken: AzureCore.CancellationToken?
+  public var dispatchQueue: Dispatch.DispatchQueue?
+  public var context: AzureCore.PipelineContext?
+  public init(clientRequestId: Swift.String? = nil, cancellationToken: AzureCore.CancellationToken? = nil, dispatchQueue: Dispatch.DispatchQueue? = nil, context: AzureCore.PipelineContext? = nil)
+}
+public class ChatClient {
+  public init(endpoint: Swift.String, credential: AzureCommunicationCommon.CommunicationTokenCredential, withOptions options: AzureCommunicationChat.AzureCommunicationChatClientOptions) throws
+  public func createClient(forThread threadId: Swift.String) throws -> AzureCommunicationChat.ChatThreadClient
+  public func create(thread: AzureCommunicationChat.CreateChatThreadRequest, withOptions options: AzureCommunicationChat.CreateChatThreadOptions? = nil, completionHandler: @escaping AzureCore.HTTPResultHandler<AzureCommunicationChat.CreateChatThreadResult>)
+  public func listThreads(withOptions options: AzureCommunicationChat.ListChatThreadsOptions? = nil, completionHandler: @escaping AzureCore.HTTPResultHandler<AzureCore.PagedCollection<AzureCommunicationChat.ChatThreadItem>>)
+  public func delete(thread threadId: Swift.String, withOptions options: AzureCommunicationChat.DeleteChatThreadOptions? = nil, completionHandler: @escaping AzureCore.HTTPResultHandler<Swift.Void>)
+  public func startRealTimeNotifications(completionHandler: @escaping (Swift.Result<Swift.Void, AzureCore.AzureError>) -> Swift.Void)
+  public func stopRealTimeNotifications()
+  public func register(event: AzureCommunicationChat.ChatEventId, handler: @escaping AzureCommunicationChat.EventHandler)
+  public func unregister(event: AzureCommunicationChat.ChatEventId)
+  @objc deinit
+}
+public struct ListChatThreadsOptions : AzureCore.RequestOptions {
+  public let maxPageSize: Swift.Int32?
+  public let startTime: AzureCore.Iso8601Date?
+  public let clientRequestId: Swift.String?
+  public let cancellationToken: AzureCore.CancellationToken?
+  public var dispatchQueue: Dispatch.DispatchQueue?
+  public var context: AzureCore.PipelineContext?
+  public init(maxPageSize: Swift.Int32? = nil, startTime: AzureCore.Iso8601Date? = nil, clientRequestId: Swift.String? = nil, cancellationToken: AzureCore.CancellationToken? = nil, dispatchQueue: Dispatch.DispatchQueue? = nil, context: AzureCore.PipelineContext? = nil)
+}
+public struct RemoveChatParticipantOptions : AzureCore.RequestOptions {
```

Comment:
Why is there no `RemoveChatParticipantsRequest`?

Reply:
When adding participants we send ChatParticipant model(s), for removing we just send the string id of the participant